tests/test_utils.py

```diff
@@ -93,7 +93,7 @@ class MyClass:
         def to_dict(self):
             return 42
 
-    assert serializer.serializer.dumps(MyClass()) == "42"
+    assert serializer.serializer.dumps(MyClass()) == b"42"
```

Comment:
In elasticsearch-py 8.x, serializers now explicitly encode the results to UTF-8. Which is more correct, after all serializing is supposed to output bytes.